projects/storefrontapp-e2e-cypress/cypress/e2e/regression/user_access/otp-login.e2e-flaky.cy.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as login from '../../../helpers/login';
+import { viewportContext } from '../../../helpers/viewport-context';
+import { interceptPost } from '../../../support/utils/intercept';
+
+export function listenForCreateVerificationToken(): string {
+  return interceptPost(
+    'createVerificationToken',
+    '/users/anonymous/verificationToken?*'
+  );
+}
+
+describe('OTP Login', () => {
+  viewportContext(['mobile'], () => {
+    describe('Create OTP', () => {
+      beforeEach(() => {
+        cy.visit('/login');
+      });
+
+      it('should be able to create a new OTP by customer click Sign In button (CXSPA-6672)', () => {
+        const user = login.registerUserFromLoginPage();
+        listenForCreateVerificationToken();
+
+        cy.log(`🛒 Logging in user ${user.email} from the login form`);
+        cy.get('cx-otp-login-form form').within(() => {
+          cy.get('[formcontrolname="userId"]').clear().type(user.email);
+          cy.get('[formcontrolname="password"]').clear().type(user.password);
+          cy.get('button[type=submit]').click();
+        });
+
+        cy.wait('@createVerificationToken')
+          .its('response.statusCode')
+          .should('eq', 201);
+
+        cy.get('cx-verification-token-form').should('exist');
+        cy.get('cx-verification-token-form').should('be.visible');
+
+        cy.log('The email being sent is asynchronous, so waiting 10s');
+        cy.wait(10000);
+
+        cy.request({
+          method: 'GET',
+          url:
+            'http://mail-ccv2.westeurope.azurecontainer.io:8025/api/v2/search?query=' +
```

Comment:
Is it safe to store this URL directly in our codebase? Perhaps it would be worthwhile to consider moving the URL to `cypress.config.ts` and setting it as an environment variable instead.

Reply:
fixed